src/org/usfirst/frc/team166/robot/OI.java

```diff
@@ -67,10 +67,13 @@ public OI() {
 
 		buttonA.whenPressed(new DriveDistance(31.4, 0.5));
 
-		if (!isReversed)
+		if (!isReversed) {
```

Comment:
Ok. I'll take this out of the code for now until I learn how to do it correctly.